test/powershell/Host/Logging.Tests.ps1

```diff
@@ -158,7 +158,7 @@ Describe 'Basic SysLog tests on Linux' -Tag @('CI','RequireSudoOnUnix') {
             }
             [string] $powershell = Join-Path -Path $PSHome -ChildPath 'pwsh'
             $scriptBlockCreatedRegExTemplate = @"
-Creating Scriptblock text \(1 of 1\):#012{0}(`u{23CE}|\?|#012)*ScriptBlock ID: [0-9a-z\-]*#012Path:.*
+Creating Scriptblock text \(1 of 1\):#012{0}(⏎|#012)*ScriptBlock ID: [0-9a-z\-]*#012Path:.*
```

Comment:
Why do we need this? `u{23CE} looks more clear for me.

Reply:
The tests fail if we use the escape sequence.

Reply:
I tried to copy and paster the code in console and lost the char. Also the tests become unreadable. I wonder why doesn't the Unicode syntax sugar work?

Reply:
@iSazonov that's going to vary by the font you use.  I worked with @rjmholt to come up with this solution.  The previous solution that was working, was actually just looking for a `?` which is less specific.

Reply:
I remember that we had Unicode chars in files and it was a headache - editors silently break them. After that we made a conclusion to keep all files in ASCII.